python/cudf/cudf/core/column/datetime.py

```diff
@@ -255,7 +255,7 @@ def binary_operator(
             return binop_offset(self, rhs, op)
         lhs, rhs = self, rhs
         if op in ("eq", "ne", "lt", "gt", "le", "ge"):
-            out_dtype = np.bool
+            out_dtype = np.dtype(np.bool_)  # type: Dtype
```

Comment:
Does this work for older versions of Numpy as well?

Reply:
Yes:

```
In [3]: np.dtype(np.bool) == np.dtype(np.bool_)
Out[3]: True

In [4]: np.__version__
Out[4]: '1.19.4'
```

`np.bool` has disappeared from numpy, so we should use either just `bool` or `np.bool_/np.bool8` going forward.

Reply:
I meant the `# type: Dtype` 😄

Reply:
That's our own type :)